bedrock/firefox/urls.py

```diff
@@ -73,7 +73,7 @@
     # This dummy page definition makes it possible to link to /firefox/ (Bug 878068)
     url('^firefox/$', views.fx_home_redirect, name='firefox'),
 
-    url('^firefox/os/$', views.firefox_os_index, name='firefox.os.index'),
+    url('^firefox/os/$', views.fxos, name="fxos"),
```

Comment:
I had no particular reason to change it. Let me make one up.... it's shorter ;) I can change it back if it is better to have it as firefox.os.index - either way, will see what @pmclanahan says.